examples/github-appcards/netlify/functions/issues.js

```diff
@@ -0,0 +1,111 @@
+/* 
+
+  This endpoint updates Miro App Cards when there is an update to their information
```

Comment:
Suggestion:

"This endpoint updates title and/or description of Miro app cards when there the corresponding data source is updated in GitHub."